tools/convert_to_ir.py

```diff
@@ -1,63 +0,0 @@
-# Copyright (C) 2020 Intel Corporation
```

Comment:
mr RewriteEverythingEachTime please update documentation as well

Reply:
This commit should not be in this PR. Sorry for that.